File: app/src/main/java/com/kota/Bahamut/Command/BahamutCommandLoadFirstBlock.java
package com.kota.Bahamut.Command;

import androidx.annotation.NonNull;

import com.kota.Bahamut.ListPage.TelnetListPage;
import com.kota.Bahamut.ListPage.TelnetListPageBlock;

public class BahamutCommandLoadFirstBlock extends TelnetCommand {
    public BahamutCommandLoadFirstBlock() {
        this.Action = LoadFirstBlock;
    }

    public void execute(TelnetListPage aListPage) {
    }

    public void executeFinished(TelnetListPage aListPage, TelnetListPageBlock aPageData) {
        setDone(true);
    }

    @NonNull
    public String toString() {
        return "[LoadFirstBlock]";
    }
}

File: app/src/main/java/com/kota/Bahamut/Command/BahamutCommandLoadFirstBlock.kt
package com.kota.Bahamut.Command

import com.kota.Bahamut.ListPage.TelnetListPage
import com.kota.Bahamut.ListPage.TelnetListPageBlock

class BahamutCommandLoadFirstBlock : TelnetCommand() {
    init {
        this.Action = BahamutCommandDefs.Companion.LoadFirstBlock
    }

    override fun execute(aListPage: TelnetListPage?) {
    }

    override fun executeFinished(aListPage: TelnetListPage?, aPageData: TelnetListPageBlock?) {
        setDone(true)
    }

    override fun toString(): String {
        return "[LoadFirstBlock]"
    }
}
